tests/e2e/support/deploy-to-netlify.ts

```diff
@@ -0,0 +1,107 @@
+import { execaCommand } from 'execa'
+import fg from 'fast-glob'
+import { copyFile, mkdir, mkdtemp, readFile } from 'node:fs/promises'
+import { tmpdir } from 'node:os'
+import { dirname, join, resolve } from 'node:path'
+import { fileURLToPath } from 'node:url'
+import { cpus } from 'os'
+import pLimit from 'p-limit'
+
+// https://app.netlify.com/sites/remix-compute-e2e-tests
+const SITE_ID = process.env.NETLIFY_SITE_ID ?? 'b1858b58-d401-4aef-9b3b-80b8ae9247c0'
+
+export interface Fixture {
+  deployId: string
+  url: string
+  logs: string
+}
+
+/**
+ * Prepare an instance of the given fixture site and deploy it to Netlify.
+ * @param fixtureName name of the folder inside the fixtures folder
+ */
+export const deployFixture = async (fixtureName: string): Promise<Fixture> => {
+  const isolatedFixtureRoot = await prepareFixture(fixtureName)
+  const result = await deploySite(isolatedFixtureRoot)
+  return result
+}
+
+/**
+ * Copy a fixture site to an isolated temp directory, prepare its dependencies, link it against any
+ * applicable local packages, and make it deployable to Netlify.
+ */
+const prepareFixture = async (fixtureName: string): Promise<string> => {
+  const isolatedFixtureRoot = await mkdtemp(join(tmpdir(), 'netlify-remix-compute-e2e-'))
+  console.log(`📂 Copying fixture '${fixtureName}' to '${isolatedFixtureRoot}'...`)
+
+  const src = fileURLToPath(new URL(`../fixtures/${fixtureName}`, import.meta.url))
+  const files = await fg.glob('**/*', {
+    ignore: ['node_modules'],
+    dot: true,
+    cwd: src,
+  })
+
+  const limit = pLimit(Math.max(2, cpus().length))
```

Comment:
Learning - why are we limiting cpus for the `mkdir` and `copyFile` commands?

Reply:
Because I [copied this from the next runtime](https://github.com/netlify/next-runtime-minimal/blob/5303726cedc2963db5e7be8ea795d9f022edc885/tests/utils/create-e2e-fixture.ts#L115) and I assumed there was a good reason!

Reply:
I've been trying to trace the origin of this and found a comment from @pieh here:

https://github.com/netlify/next-runtime/pull/2104/files#r1197942689

Can you shed more light on this, Michal?

Reply:
For the size of fixtures we have this is probably not necessary here (given that we skip copying `node_modules`). It also isn't as crucial nowadays with SSDs being used commonly (hard drives had much worse perf handling concurrent operations due to actual physical design)

Overall this is just good practice to put some limits for concurrent work when size of work and actual hardware it will run on is ~unknown (open file handles limit, concurrent operations starving to have actual access to resources to complete it's task), so it's best summarised with "might not be needed, but it doesn't hurt to do it".